extensions/scarb-snforge-test-collector/src/compilation.rs

```diff
@@ -0,0 +1,44 @@
+use anyhow::Result;
+use cairo_lang_sierra::program::{ProgramArtifact, VersionedProgram};
+use rayon::prelude::{IntoParallelRefIterator, ParallelIterator};
+use serde::Serialize;
+
+use crate::compilation::test_collector::{collect_tests, TestCaseRaw};
+use crate::crate_collection::{CrateLocation, TestCompilationTarget};
+use crate::metadata::CompilationUnit;
+
+pub mod test_collector;
+
+#[derive(Debug, Clone, Serialize, PartialEq)]
+pub struct CompiledTestCrateRaw {
```

Comment:
Got rid of the generics to match in https://github.com/foundry-rs/starknet-foundry/pull/1157